TemplePlus/feat.cpp

```diff
@@ -259,10 +260,26 @@ void LegacyFeatSystem::_GetNewFeatsFromFile()
 					
 					if (prereqArgCount < prereqCount)
 					{
-						featSpec.prereqs[prereqArgCount++].featPrereqCodeArg = atol(ch);
+						if (featSpec.prereqs.capacity() <= prereqArgCount)
```

Comment:
Are you sure you wanted to use capacity() and not size() here? size() is the real current size of the vector.